schematic/schemas/commands.py

```diff
@@ -114,35 +126,44 @@ def convert(
                 for warning in war:
                     logger.warning(warning)
 
+    if output_path:
+        output_jsonld = output_path
+
+    if output_jsonld is None:
+        output_file_no_ext = re.sub("[.](jsonld|csv|pickle)$", "", schema)
+    else:
+        output_file_no_ext = re.sub("[.](jsonld|csv|pickle)$", "", output_jsonld)
+
+    logger.info(
+        "By default, the JSON-LD output will be stored alongside the first "
+        f"input CSV or JSON-LD file. In this case, it will appear here: '{output_jsonld}'. "
+        "You can use the `--output_jsonld` argument to specify another file path."
+    )
+
+    if output_type in ["graph", "all"]:
+        logger.info("Export graph to pickle.")
+        output_graph = output_file_no_ext + ".pickle"
+        export_graph(graph_data_model, output_graph)
+        if output_type == "graph":
+            click.echo(f"Graph created {output_graph}")
+            return 0
```

Comment:
You should not hard coded the return code

Reply:
Since there is an expectation that the code stops executing at this line, what is the alternative here?

From this api doc:
https://click.palletsprojects.com/en/8.0.x/api/#commands

It looks like the expectation is spelled out in the `standalone_mode` parameter:
```
standalone_mode – the default behavior is to invoke the script in standalone mode. Click will then handle exceptions and convert them into error messages and the function will never return but shut down the interpreter. If this is set to False they will be propagated to the caller and the return value of this function is the return value of [invoke()](https://click.palletsprojects.com/en/8.0.x/api/#click.BaseCommand.invoke).
```

So if I am reading this correctly just a `return` with no value is sufficient here to stop execution?

Reply:
Yeah you are right. I was under the impression that even without writing  `return=0`, the success code will be 0. Even though we don't have to explicitly write `return=0`, returning 0 explicitly in the success case makes the code more consistent and predictable. Thanks for pointing that out.

Reply:
I am going to mark this as resolved but let me know if I'm missing something. I tried removing the other `return 0` above, but mypy did not like that `error: Return value expected  [return-value]`

Reply:
@afwillia mypy didn't like it because you have updated the return type of the function from `None` to `int`. If you set it back to `None` it would allow you to do a `return` with no values.